src/components/Footer/Footer.scss

```diff
@@ -15,36 +15,51 @@
   padding: 0 5px;
 
   select {
-    height: 20px;
-    font-size: $footer-font-size;
-    border-radius: 0;
+    display: none; // Hide select from react-spatial MousePosition
+  }
+
+  input {
+    /*
+     * Hide dummyInput from react-select
+     * before issue resolved: https://github.com/JedWatson/react-select/issues/3995
+     */
+    position: absolute !important;
+    top: 0;
```

Comment:
why not display: none?

Reply:
if I remove this css I don't see it.

Reply:
If I add display:none, the input is not clickable anymore to open the list of projections.

Reply:
Yes for the display none. but If you remove this css it works for me,